torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -505,6 +504,16 @@ def test_send_remote_module_over_the_wire(self):
                 dst_worker2_name, remote_forward_async, (remote_module, args)
             )
             self.assertEqual(ret2, tuple(reversed(args)))
+            
```

Comment:
Can you move these to 2 new test cases above `test_unsupported_methods`? It's better to have different test cases specific to different functionality. You can see that `get_module_rref` and `remote_parameters()` methods are tested separately.